addons/l10n_in_hr_payroll/report/payslip_report_view.xml

```diff
@@ -84,7 +84,7 @@
         <field name="groups_id" eval="[(4,ref('hr_payroll.group_hr_payroll_user'))]"/>
         <field name="arch" type="xml">
             <data>
-                <xpath expr="//div[contains(@class, 'manage_reports')]" position="inside">
+                <xpath expr="//div[hasclass('manage_reports')]" position="inside">
```

Comment:
If it was intended it's misleading as it was for `o_project_kanban_box` . I'll fix it.